core/tsparticles/src/Core/Loader.ts

```diff
@@ -103,7 +103,7 @@ export class Loader {
         let canvasEl: HTMLCanvasElement;
         let generatedCanvas: boolean;
 
-        if (domContainer.tagName === "canvas") {
+        if (domContainer.tagName === "CANVAS") {
```

Comment:
I think .toLower() is a cleaner solution

Reply:
I changed it, but tastes differ I guess :smile:.